src/main/build/build_config.h

```diff
@@ -66,6 +66,7 @@ typedef enum {
     MCU_TYPE_RP2350A,
     MCU_TYPE_RP2350B,
     MCU_TYPE_GD32F460,
+    MCU_TYPE_H735,
```

Comment:
This is already in master. Please rebase branch against master to update.

https://github.com/betaflight/betaflight/blob/60a91125409ff5bca9a8738aa3e716e46182d5e0/src/main/build/build_config.h#L66-L68